etna/models/nn/deepar_new/deepar.py

```diff
@@ -0,0 +1,372 @@
+            sample["encoder_real"][:, 0] = values_real[start_idx + 1 : start_idx + encoder_length, 0] / sample["weight"]
+            sample["decoder_real"][:, 0] = (
+                values_real[start_idx + encoder_length : start_idx + total_sample_length, 0] / sample["weight"]
+            )
+
+            return sample
+
+        start_idx = 0
+        while True:
+            batch = _make(
+                values_target=values_target,
+                values_real=values_real,
+                segment=segment,
+                start_idx=start_idx,
+                encoder_length=encoder_length,
+                decoder_length=decoder_length,
+            )
+            if batch is None:
+                break
+            yield batch
+            start_idx += 1
+
+    def configure_optimizers(self) -> "torch.optim.Optimizer":
+        """Optimizer configuration."""
+        optimizer = torch.optim.Adam(self.parameters(), lr=self.lr, **self.optimizer_params)
+        return optimizer
+
+
+class DeepARModelNew(DeepBaseModel):
+    """DeepAR based model on LSTM cell.
+
+    Note
+    ----
+    This model requires ``torch`` extension to be installed.
+    Read more about this at :ref:`installation page <installation>`.
+    """
+
+    def __init__(
+        self,
+        input_size: int,
+        encoder_length: int,
+        decoder_length: int,
+        num_layers: int = 2,
+        hidden_size: int = 16,
+        lr: float = 1e-3,
+        scale: bool = True,
+        n_samples: int = 1,
+        loss: Optional[Union[GaussianLoss, NegativeBinomialLoss]] = None,
+        train_batch_size: int = 16,
+        test_batch_size: int = 16,
+        optimizer_params: Optional[dict] = None,
+        trainer_params: Optional[dict] = None,
+        train_dataloader_params: Optional[dict] = None,
+        test_dataloader_params: Optional[dict] = None,
+        val_dataloader_params: Optional[dict] = None,
+        split_params: Optional[dict] = None,
+    ):
+        """Init DeepAR model based on LSTM cell.
+
+        Parameters
+        ----------
+        input_size:
+            size of the input feature space: target plus extra features
+        encoder_length:
+            encoder length
+        decoder_length:
+            decoder length
+        num_layers:
+            number of layers
+        hidden_size:
+            size of the hidden state
+        lr:
+            learning rate
+        scale:
+            if True, scale target values in batch before training by :math:`1 + mean`, where :math:`mean` is mean of target values in the encoder part of batch
+        n_samples:
+            if 1, return theoretical mean of distribution as a predicted value. if greater than 1, return mean of `n_samples` generated from Monte Carlo sampling
+        loss:
+            loss function
+                * **:py:class:`etna.models.nn.deepar_new.GaussianLoss`**: use Gaussian distribution for counting loss
```

Comment:
It is rendered  srangly, may be because of "**", try without them and ":" in the end
```
:py:class:`etna.models.nn.deepar_new.loss.GaussianLoss`
```
<img width="635" alt="Снимок экрана 2023-11-15 в 08 59 47" src="https://github.com/etna-team/etna/assets/55380696/4665dde0-5e4f-45f4-aeb0-14fcf5fb9df8">